device-types/Ubiquiti/UVC-G6-Bullet.yaml

```diff
@@ -0,0 +1,89 @@
+---
+manufacturer: Ubiquiti
+model: Unifi G6 Bullet
+slug: ubiquiti-unifi-uvc-g6-bullet
+part_number: UVC-G6-Bullet
+comments: |
```

Comment:
Thank you for your contribution,
Please reduce the amount of information in the comments property, this should only be a brief description with a link to the documentation.